src/Domain/Configuration.php

```diff
@@ -126,6 +132,14 @@ public static function getAcceptedRequirements(): array
         return self::ACCEPTED_REQUIREMENTS;
     }
 
+    /**
+     * @return array<string>
+     */
+    public static function getAcceptedFormats(): array
```

Comment:
this function is only used here, so it should be private. But regarding it's content, we could even drop it :+1: